supplementary_style_guide/glossary_terms_conventions/product_conventions/red-hat-single-sign-on.adoc

```diff
@@ -0,0 +1,146 @@
+
+[discrete]
+[[authorization]]
+== Authorization
+The process of granting access to a user.
+
+[discrete]
+[[client-adapters]]
+== Client adapters
+Client adapters are plugins that you install into your application environment to be able to communicate and be secured by Red Hat Single Sign-On. Red Hat Single Sign-On has a number of adapters for different platforms that you can download. There are also third-party adapters you can use for environments that Red Hat do not cover.
+
+
+[discrete]
+[[client-role]]
+== Client role
+Clients can define roles that are specific to them. This is basically a role namespace dedicated to the client.
+
+[discrete]
+[[client-scopes]]
+== Client scopes
+When a client is registered, you must define protocol mappers and role scope mappings for that client. It is often useful to store a client scope, to make creating new clients easier by sharing some common settings. This is also useful for requesting some claims or roles to be conditionally based on the value of `scope` parameter. Red Hat Single Sign-On provides the concept of a client scope for this.
+
+[discrete]
+[[clients]]
+== Clients
+Clients are entities that can request Red Hat Single Sign-On to authenticate a user. Most often, clients are applications and services that want to use Red Hat Single Sign-On to secure themselves and provide a single sign-on solution. Clients can also be entities that only want to request identity information or an access token so that they can securely invoke other services on the network that are secured by Red Hat Single Sign-On.
+
+[[composite-roles]]
+== Composite roles
+A composite role is a role that can be associated with other roles. For example a `superuser` composite role can be associated with the `sales-admin` and `order-entry-admin` roles. If a user is mapped to the `superuser` role they also inherit the `sales-admin` and `order-entry-admin` roles.
+
+[discrete]
+[[consent]]
+== Consent
+Consent is when you as an `admin` want a user to give permission to a client before that client can participate in the authentication process. After a user provides their credentials, Red Hat Single Sign-On opens a screen identifying the client requesting a login and what identity information is requested of the user. Users can decide whether or not to grant the request.
+
+[discrete]
+[[credentials]]
+== Credentials
+Credentials are pieces of data that Red Hat Single Sign-On uses to verify the identity of a user. Some examples are passwords, one-time-passwords, digital certificates, or even fingerprints.
+
+[discrete]
+[[direct-grant]]
+== Direct grant
+A way for a client to obtain an access token on behalf of a user through a REST invocation.
+
+[discrete]
+[[events]]
+== Events
+Events are audit streams that admins can view and hook into.
```

Comment:
```suggestion
Events are audit streams that admins view and connect.
```

Note: not sure if "hook into" means connect or be connected to. Please reword as originally intended.